docs/source/app_developers_guide/environment_setup.rst

```diff
@@ -67,17 +74,24 @@ Open up a terminal and run the following:
 
       C:\> git clone https://github.com/hyperledger/sawtooth-core.git
       --config core.autocrlf=false
+=======
+Download the docker compose file `here <./sawtooth-default.yaml>`_.
+>>>>>>> App Dev Guide should not use git or repo
 
 
 Environment Startup
 -------------------
 
-To start up the environment, run:
+To start up the environment, perform the following tasks:
+
+1. Open a terminal window.
+2. Change your working directory to the same directory where you saved the
```

Comment:
Might want to include and example of what you mean here. Something like "($ cd ~/Downloads)"